packages/web-runtime/src/components/VersionCheck.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div v-if="isLoading" class="version-check-loading flex items-center">
+    <span v-text="$gettext('Checking for updates')" />
+    <oc-spinner class="ml-1" size="xsmall" />
+  </div>
+  <template v-else>
```

Comment:
Even with the `template` tag I don't think that it improves readability. :-P But fine by me to keep it like it is.